### PR TITLE
Automate regional counts

### PR DESCRIPTION
**Pull request by liamtoozer**
# Motivation and Context

We need to automate regional counts for Census Rehearsal so we can grab the counts instead of having to run manual queries against the db.

# What has changed

* Added new query to gather regional counts
* Moved common SQL bits to helper function

# Links

https://trello.com/c/9e7YcwOO/1246-automate-regional-counts-3

### CODE REVIEW

**Comment by liamtoozer** [comment]
[Automate regional counts (3)](https://trello.com/c/9e7YcwOO/1246-automate-regional-counts-3)

File: regional_counts.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+if __name__ == "__main__":
+    sql_query = """
+SELECT CASE LEFT(region, 1)
+           WHEN 'E' THEN 'England'
+           WHEN 'W' THEN 'Wales'
+           WHEN 'N' THEN 'Northern Ireland'
+           ELSE region END AS region, COUNT(*)
+FROM casev2.cases
+WHERE receipt_received = true
+-- AND action_plan_id = '432f0597-0076-4adb-834b-bf249dc06ded'
```
Comment (LukeLoze1) [inline regional_counts.py]: remove commented out sql

Comment (liamtoozer) [reply regional_counts.py]: Good spot, I was testing this locally 👍

**Review by ryangrundy7 — APPROVED** [review]
LGTM 👍

**Review by LukeLoze1 — APPROVED** [review]

**Review by gavinleeedwards — CHANGES_REQUESTED** [review]
I've just run the query in prod and it's splitting the England count - can't see why though!

**Review by AdeHarrison — APPROVED** [review]

**Review by gavinleeedwards — APPROVED** [review]
Looks good and all works fine in dev GCP environment 👍

**Pull request merged**